whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MimeReportInfo.java

```diff
@@ -0,0 +1,33 @@
+package net.ripe.db.whois.api.mail.dequeue;
+
+import jakarta.mail.internet.MimeMessage;
+
+public class MimeReportInfo {
```

Comment:
Class not used?

Reply:
yes, I was using this class to get the relevant information from the part of the mail. Since we are not using the parsing that I created this is not longer used